pkg/services/provisioning/provisioning.go

```diff
@@ -21,6 +22,7 @@ type ProvisioningService interface {
 	registry.BackgroundService
 	RunInitProvisioners() error
 	ProvisionDatasources() error
+	ProvisionChannelRules(store *sqlstore.SQLStore) error
```

Comment:
Enterprise needs an update due to this change I believe.

Reply:
Enterprise should be updated after merge right?

Reply:
Yes, but could be good having those changes ready before merging this to limit enterprise builds failing.